src/ServiceControl.Monitoring/Settings.cs

```diff
@@ -0,0 +1,37 @@
+namespace ServiceControl.Monitoring
+{
+    using System;
+    using System.IO;
+    using NLog;
+    using NServiceBus;
+
+    public class Settings
+    {
+        const string DEFAULT_ENDPOINT_NAME = "Particular.ServiceControl.Monitoring";
+        public string EndpointName { get; set; } = DEFAULT_ENDPOINT_NAME;
+        public string TransportType { get; set; }
+        public string TransportConnectionString { get; set; }
+        public string LogPath { get; set; }
+        public LogLevel LogLevel { get; set; }
+        public string Username { get; set; }
+        public bool EnableInstallers { get; set; }
+
+        internal static Settings Load(SettingsReader reader)
+        {
+            var settings = new Settings
+            {
+                TransportType = reader.Read("Transport/Type", typeof(MsmqTransport).AssemblyQualifiedName),
+                TransportConnectionString = reader.Read<string>("Transport/ConnectionString"),
+                LogPath = CalculateLogPathForEndpointName(DEFAULT_ENDPOINT_NAME),
```

Comment:
Just to point out, it was set to override based on the endpoint name when passed in at the command line. Since your comment (which I missed, apologies) Mike refactor it to call the method with the endpoint name when the LogPath property is accessed.